### DISCUSSION TITLE
closechannel feerateSatByte action not as expected

### DISCUSSION THREAD

**Discussion by PeterRounce** [ACINQ/phoenixd#129] [General]
Using [closechannel](https://phoenix.acinq.co/server/api#close-channel) from the CLI with a feerateSatByte of 10 actioned the channel close with a fee rate of 4.

I expected it to match the fee rate that I gave in the CLI command, so this was not as expected and I cannot find any documentation that describes the behaviour in any more detail.

**Comment by pm47** [comment]
Hello, this operates at the lightning protocol level. In the `close` case, the requested feerate is negotiated between the two peers, because costs are shared.

An upcoming new [`simple_close`](https://github.com/lightning/bolts/pull/1205) feature to the protocol will allow you to set the exact feerate in exchange of paying the full mining fee.

Feel free to share your channel id or node id to phoenix@acinq.co and we will bump the transaction.
